app/src/main/res/values/strings.xml

```diff
@@ -842,6 +842,15 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
 
+    <string name="quest_camp_drinking_water_title">Is drinking water freely available to customers here?</string>
```

Comment:
```suggestion
    <string name="quest_camp_drinking_water_title">Is drinking water freely available to campers here?</string>
```

Customers sounds odd in the context of accommodation. I've checked with en-US and en-GB dictionaries - camper is fine to refer to a person that camps (it can also mean the caravan/motorhome, but context is clear here)